sdk/js/src/types.ts

```diff
@@ -41,6 +41,7 @@ export type initialize = (
 export interface DVCOptions {
     flushEventsMS?: number
     reactNative?: boolean
+    enableCloudEntityData?: boolean
```

Comment:
I'm wondering if there's a better name for this option, maybe `enableCloudUserData`? I'll consult with the team